channels_fields/serializers.py

```diff
@@ -280,23 +257,16 @@ def create(self, validated_data):
             self.upsert_subfields(field_channel, validated_data)
             return field_channel
 
+    def to_representation(self, data):
+        return FieldChannelSerializer(context=self.context).to_representation(data)
```

Comment:
My goal here was to make sure that the create/update responses were the same same as the GET responses. This is accurately reflected in our OpenAPI schema with some minor manual decoration.